src/components/UI/loading_indicator.js

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
+
+const LoadingIndicator = props => (
+  <div className='progress_button--status-indicator'>
```

Comment:
@oponder This isn't done yet, is it? Having the class `progress_button--status-indicator` in use here instead of CSS in JS.

Reply:
Right, the goal is to get rid of that where possible. Maybe it's not possible yet here for some reason.

ping @ograu, I didn't notice this when I approved it :P any reason we're not getting the css-in-js for these classes here?

Reply:
@oponder IDK, sorry, I myself am wondering same thing, I am going to fix it now